ports/openfx/Openfx.cmake

```
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 3.20)
+
+project(openfx VERSION 1.4.0 LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+if(WIN32)
+    add_compile_definitions(WINDOWS)
+    add_compile_options(/DNOMINMAX)
+    add_definitions(-DWIN64)
+    set(OS_VAR "windows")
+    set(OFX_ARCH_NAME "Win64")
+endif()
+
+set(OFX_HEADERS_DIR ${CMAKE_CURRENT_SOURCE_DIR}/include CACHE INTERNAL "OFX_HEADERS_DIR")
+
+add_library(OpenFx INTERFACE)
+target_include_directories(OpenFx
+    INTERFACE
+    $<BUILD_INTERFACE:${OFX_HEADERS_DIR}>
+    $<INSTALL_INTERFACE:include/openfx>
+)
+
+add_subdirectory(Support)
+
+install(
+    TARGETS OpenFx OfxSupport
+    EXPORT openfx-export
+    LIBRARY DESTINATION lib
+    ARCHIVE DESTINATION lib
+)
+
+install(
+    EXPORT openfx-export
+    FILE unofficial-openfxConfig.cmake
+    NAMESPACE unofficial::OpenFx::
```

I assume you want the `....OpenFX::OpenFX`, but OTOH we may want `unofficial::<port>::...`.
```suggestion
    NAMESPACE unofficial::openfx::
```